### PR TITLE
Tidy printing.preview slightly

### PR DESCRIPTION
**Pull request by eric-wieser**
#### Brief description of what is fixed or changed
See commit messages. Minor cleanup, no behavior changes.

#### Release Notes

NO ENTRY

### CODE REVIEW

File: sympy/printing/preview.py [hunk 1]
```diff
@@ -279,77 +330,16 @@ def preview(expr, output='png', viewer=None, euler=True, packages=(),
                 outputbuffer.write(fh.read())
         elif viewer == "pyglet":
             try:
-                from pyglet import window, image, gl
-                from pyglet.window import key
+                import pyglet
```
Comment (eric-wieser) [inline sympy/printing/preview.py:333]:
```suggestion
                import pyglet  # noqa: F401
```

**Comment by eric-wieser** [comment]
Once this goes in, I have a follow-up which makes pdf previewing work on windows

**Comment by oscarbenjamin** [comment]
This looks like an improvement. I have previously noticed that this code was a mess but I considered that it was probably best to remove the pyglet preview code altogether and just use matplotlib instead (#2061 and https://github.com/sympy/sympy/issues/8446#issuecomment-557798066). What are you thinking of for the general direction here?

**Comment by eric-wieser** [comment]
The follow up was going to be to have a mode which calls `xdg-open` / `open` / `os.startfile` to let the operating system choose a suitable image / pdf viewer

**Comment by oscarbenjamin** [comment]
> a mode which calls `xdg-open` / `open` / `os.startfile` to let the operating system choose a suitable image / pdf viewer

If we had that then we could deprecated and eventually remove the pyglet viewer (I think we should).

This looks good though so I'll merge this now.

**Pull request merged**